test/constants.ts

```diff
@@ -29,7 +29,9 @@ export const ELEVEN_YEARS_FROM_NOW_IN_SECONDS = Math.round(
 
 export const ONE = utils.parseUnits("1", 18);
 export const WAD = utils.parseUnits("1", 18);
-
+export const SQRT_MAX_UINT256 = BigNumber.from(
+  "340282366920938463463374607431768211456"
```

Comment:
Nice. I ran through our test suite as well with SQRT_MAX_UINT256.sub(1) as maxSupply for bond and nothing explicitly broke (except for rounding errors)